ibis/backends/base/sql/alchemy/datatypes.py

```diff
@@ -175,7 +175,7 @@ def _pg_array(dialect, itype):
 @to_sqla_type.register(Dialect, dt.Struct)
 def _struct(dialect, itype):
     return StructType(
-        [(name, to_sqla_type(dialect, type)) for name, type in itype.pairs.items()]
+        [(name, to_sqla_type(dialect, type)) for name, type in itype.fields.items()]
```

Comment:
Can we get rid of `fields` entirely and rely on the `Mapping` interface? Doing that can definitely be a follow-up PR.

Reply:
I wasn't entirely sure whether we want that, but we already provide most of the mapping API for schema, so I guess we should do the same with StructType. Going to do that in a follow-up.